document-warehouse/package.json

```diff
@@ -7,7 +7,9 @@
         "test": "c8 mocha test/*.js --timeout 600000"
     },
     "dependencies": {
-        "@google-cloud/contentwarehouse": "^0.5.1",
-        "assert": "^2.0.0"
+        "@google-cloud/contentwarehouse": "^0.5.1"
+    },
+    "devDependencies": {
```

Comment:
Not sure why this is needed?

Reply:
This defined the version for assert in the test directory.